### PR TITLE
swap search modal components for doorway-ui

### PR DESCRIPTION
**Pull request by isaac-looker**
# Pull Request Template

## Issue Overview

This PR addresses refactor/restyle search modal. This PR does no restyling; it only clones the logic, using the existing components in doorway-ui. 

- [ ] This change addresses the issue in full
- [x] This change addresses only certain aspects of the issue
- [ ] This change is a dependency for another issue
- [ ] This change has a dependency from another issue

## Description

Please include a summary of the change and which issue(s) is addressed. Please also include relevant motivation and context. List any dependencies that are required for this change.

## How Can This Be Tested/Reviewed?

Provide instructions so we can review.

Describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration.

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have added QA notes to the issue with applicable URLs
- [ ] I have performed a self-review of my own code
- [ ] I have reviewed the changes in a desktop view
- [ ] I have reviewed the changes in a mobile view
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have assigned reviewers
- [ ] I have run `yarn generate:client` and/or created a migration if I made backend changes that require them
- [ ] My commit message(s) is/are polished, and any breaking changes are indicated in the message and are well-described
- [ ] Commits made across packages purposefully have the same commit message/version change, else are separated into different commits

## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.

### CODE REVIEW

**Comment by phattya** [comment]
Isaac this is getting there!!! Great progress!!!  I built & tested it and functionality seems to be working well. I'll be able to dive into the code a bit later, but wanted to point out that we're missing Rent minimum, right now. Should be a max & min.

**Comment by isaac-looker** [comment]
> Isaac this is getting there!!! Great progress!!! I built & tested it and functionality seems to be working well. I'll be able to dive into the code a bit later, but wanted to point out that we're missing Rent minimum, right now. Should be a max & min.

added min rent field

File: sites/public/src/components/listings/search/ListingsSearchModal.tsx [hunk 1]
```diff
@@ -145,9 +172,18 @@ export function ListingsSearchModal(props: ListingsSearchModalProps) {
...
+        <input
+          type="text"
+          name="minRent"
+          value={formValues.minRent}
+          placeholder="Min Price: $"
+          style={textInputStyle}
+          onChange={(e: React.FormEvent<HTMLInputElement>) => {
+            updateValue("minRent", e.currentTarget.value)
+          }}
+        />
```
Comment (chriscasto) [inline sites/public/src/components/listings/search/ListingsSearchModal.tsx:186]: There should be a hyphen in between these

Comment (isaac-looker) [reply sites/public/src/components/listings/search/ListingsSearchModal.tsx:186]: I'll add this in a future restyling PR.

**Review by chriscasto — APPROVED** [review]
I noticed that "San Francisco" is always set even if only one other county is checked.

image

This actually results in some results being returned in my local instance.

image

I don't want to block on any of this since the core functionality is mostly there and we're running low on time, but I think it's worth addressing this and a couple other things I called out in other comments.

File: sites/public/src/components/listings/search/ListingsSearchModal.tsx [hunk 2]
```diff
@@ -162,11 +198,11 @@ export function ListingsSearchModal(props: ListingsSearchModalProps) {
 
       <div style={inputSectionStyle}>
         <div>Locations</div>
-        <MultiSelectGroup
+        <FieldGroup
```
[the remark this answers is not in the document]
Comment (isaac-looker) [reply sites/public/src/components/listings/search/ListingsSearchModal.tsx:201]: I don't see anything like that immediately; I'm hoping we can use `flex`.

**Comment by isaac-looker** [comment]
> I noticed that "San Francisco" is always set even if only one other county is checked.
> 
> image
> 
> This actually results in some results being returned in my local instance.
> 
> image
> 
> I don't want to block on any of this since the core functionality is mostly there and we're running low on time, but I think it's worth addressing this and a couple other things I called out in other comments.

fixed SF showing up

File: sites/public/src/components/listings/search/ListingsSearchCombined.tsx [hunk 3]
```diff
@@ -207,7 +207,8 @@ const locations: FormOption[] = [
   {
     label: "San Francisco",
```
Comment (ifranch) [inline sites/public/src/components/listings/search/ListingsSearchCombined.tsx:208]: Can you add Napa to this list as well?

File: sites/public/src/components/listings/search/ListingsSearchModal.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (ifranch) [reply sites/public/src/components/listings/search/ListingsSearchModal.tsx:204]: Yep this should be fairly straightforward to style

**Review by ifranch — APPROVED** [review]
I tested and confirmed that SF listings don't show anymore by default, thanks for the fix. Functionality looks good to me!

**Review by phattya — APPROVED** [review]
Lgtm & functioning well when I test. Thanks @isaac-looker !!!

**Pull request merged**